packages/compiler/plugin.ts

```diff
@@ -57,11 +61,13 @@ export const intro = (options: Options) => {
   }
 };
 
-export const unplugin = createUnplugin((options: Options = {}) => {
+// eslint-disable-next-line @typescript-eslint/default-param-last
+export const unplugin = createUnplugin((options: Options = {}, meta) => {
```

Comment:
IMHO you shouldn't disable the `@typescript-eslint/default-param-last` rule here. Please provide a default value for `meta`.

Reply:
The meta argument will be injected by unplugin so it's never undefined.
I agree with the goal of the rule when creating API but here we don't control the API.
I can change to this to avoid the default on first argument:
```suggestion
export const unplugin = createUnplugin((_options?: Options, meta) => {
  const options = _options ?? {}
```